src/test/scala/uk/gov/hmrc/perftests/ecl/requests/ReturnsRequests.scala

```diff
@@ -24,7 +24,7 @@ import uk.gov.hmrc.perftests.ecl.requests.AuthRequests._
 
 object ReturnsRequests extends Configuration {
 
-  val startUrl: String                              = s"$returnAuthWizardUrl/period/22XY"
+  val startUrl: String                              = s"$returnAuthWizardUrl/period/23XY"
```

Comment:
is this going to fail next year for example because the financial year will be 24XX?

Reply:
done